doc/api/cli.md

```diff
@@ -97,6 +97,43 @@ Automatically zero-fills all newly allocated [Buffer][] and [SlowBuffer][]
 instances.
 
 
+### `--preserve-symlinks`
+
+Instructs the module loader to preserve symbolic links when resolving and
+caching modules.
+
+By default, when Node.js loads a module from a path that is symbolically linked
+to a different on disk location, Node.js will resolve the link and use the
```

Comment:
Maybe say "will dereference the link" instead of "resolve". Using the "resolve" language has been a source of confusion because it means to change a relative path to an absolute one -- which has nothing to do with dereferencing a symlink.

A symlink is a file that could be represented with a relative path or an absolute one just like any other file -- and if a relative path is used, it could be "resolved" to an absolute one without any dereferencing. I don't think merely mentioning "real path" is sufficient (but it's good, don't remove it). I think we should use "dereference" language here and anywhere else related in the future, IMO.